several fixes (raduino_v1.0.6), see README.md for revision record

amunters opened the pull request:
Hi Ashhar,

here is the same sketch I sent to the BITX20 group on iogroups earlier today.
If you agree, please merge with your bitx40 master. So that the rest of the bitx40 community can use it as a 'standard' raduino code.

Thanks, 73
Allard PE1NWL

amunters commented:
Hi Ashhar,
here is another update (raduino_v1.0.3) which includes the "flutter fix" for improved tuning as suggested by Jerry KE7ER.
It gives a much smoother tuning without flutter.
Please review and merge with your branch. Thanks, 73 Allard PE1NWL

gauravjee commented:
Hi 
I am new here and just got my first bitx 40
Can I update the code files directly downloaded from here in to my raudino using IDE application

Thanks

amunters commented:
Go to the branch at https://github.com/amunters/bitx40 which includes the latest updates (currently raduino_v1.05).
Download from there and open it in your IDE. From your IDE you can compile and upload it to your raduino.
Also be sure to use the correct si5351 library v2.01 (see README.md).

amunters commented:
Hi Ashhar,
the list of fixes and enhancements of my pull request has grown, see README.md for a complete overview.
I'm now still working on the Function Button improvement (select VFO A/B, USB/LSB, RIT, Calibration, etc  via the FBUTTON), almost ready now.
73, Allard PE1NWL

amunters committed:
Update README.md

amunters committed:
Update README.md

amunters committed:
Update README.md

amunters committed:
Update README.md

amunters committed:
Update README.md

amunters committed:
Update README.md

amunters committed:
Update README.md

amunters committed:
Update README.md

amunters committed:
Update README.md

amunters committed:
Update README.md

amunters committed:
Update README.md

amunters committed:
Update and rename raduino_v1.15.ino to raduino_v1.15.1.ino

amunters committed:
Update operating instructions

amunters committed:
Update README.md

amunters committed:
Update and rename raduino_v1.15.1.ino to raduino_v1.16.ino

amunters committed:
Add files via upload

amunters committed:
Update operating instructions

amunters committed:
Update operating instructions

amunters committed:
Update README.md

amunters committed:
Update README.md

amunters committed:
Update operating instructions

amunters committed:
Update and rename raduino_v1.16.ino to raduino_v1.17.ino

amunters committed:
Update README.md

amunters committed:
Update operating instructions

amunters committed:
Update and rename raduino_v1.17.ino to raduino_v1.17.1ino

amunters committed:
Update README.md

amunters committed:
Update operating instructions

amunters committed:
Rename raduino_v1.17.1ino to raduino_v1.17.1.ino

amunters committed:
Update and rename raduino_v1.17.1.ino to raduino_v1.18.ino

amunters committed:
Update operating instructions

amunters committed:
Update README.md

amunters committed:
Update operating instructions

amunters committed:
Update and rename raduino_v1.18.ino to raduino_v1.19.ino

amunters committed:
Update operating instructions

amunters committed:
Update README.md

amunters committed:
Update raduino_v1.19.ino

amunters committed:
Update and rename raduino_v1.19.ino to raduino_v1.20.ino

amunters committed:
Update operating instructions

amunters committed:
Update README.md

amunters committed:
Update README.md

amunters committed:
Update and rename raduino_v1.20.ino to raduino_v1.21.ino

amunters committed:
Rename raduino_v1.21.ino to raduino_v1.20.1.ino

amunters committed:
Update README.md

amunters committed:
Update operating instructions

amunters committed:
Update and rename raduino_v1.20.1.ino to raduino_v1.21.ino

amunters committed:
Update operating instructions

amunters committed:
Add files via upload

amunters committed:
Update README.md

amunters committed:
Update README.md

amunters committed:
Update operating instructions

amunters committed:
Update and rename raduino_v1.21.ino to raduino_v1.22.ino

amunters committed:
Update raduino_v1.22.ino

amunters committed:
Update README.md

amunters committed:
Update operating instructions

amunters committed:
Update and rename raduino_v1.22.ino to raduino_v1.23.ino

amunters committed:
Update operating instruction for v1.23

amunters committed:
Update README.md

amunters committed:
Update README.md

amunters committed:
Update and rename raduino_v1.23.ino to raduino_v1.23.1.ino

amunters committed:
Update README.md

amunters committed:
Update and rename raduino_v1.23.1.ino to raduino_v1.24.ino

amunters committed:
updated for v1.24

amunters committed:
Update README.md

amunters committed:
Update README.md

amunters committed:
Update README.md

amunters committed:
Add files via upload

capacitive touch keyer modification

amunters committed:
Update and rename raduino_v1.24.ino to raduino_v1.25.ino

amunters committed:
Update operating instructions

amunters committed:
Update README.md

amunters committed:
Add files via upload

amunters committed:
Update and rename raduino_v1.25.ino to raduino_v1.25.1.ino

amunters committed:
Update operating instructions

amunters committed:
Update README.md

amunters committed:
Update README.md

amunters committed:
Update operating instructions

amunters committed:
Add files via upload

amunters committed:
Update README.md

amunters committed:
Update operating instructions

amunters committed:
Update and rename raduino_v1.25.1.ino to raduino_v1.26.ino

amunters committed:
Update operating instructions

amunters committed:
Update README.md

amunters committed:
Update operating instructions

amunters committed:
Update and rename raduino_v1.26.ino to raduino_v1.27.ino

amunters committed:
Update operating instructions

amunters committed:
Update README.md

amunters committed:
Update README.md

amunters committed:
Update README.md

mumrah committed:
Convert "operating instructions" to markdown.

Also rename the file to operating-instructions.md so Github detects it as
Markdown properly.

mumrah committed:
Tidy up the README.md a bit

amunters committed:
Merge pull request #9 from mumrah/op-instructions-markdown-squash

Convert "operating instructions" to markdown.

amunters committed:
Merge pull request #10 from mumrah/cleanup-readme-squash

Tidy up the README.md a bit

amunters committed:
Update README.md

amunters committed:
Update README.md

amunters committed:
Update README.md

amunters committed:
Update operating-instructions.md

amunters committed:
Update operating-instructions.md

amunters committed:
Update and rename raduino_v1.27.ino to raduino_v1.27.1.ino

amunters committed:
Update README.md

amunters committed:
Update README.md

amunters committed:
Update and rename raduino_v1.27.1.ino to raduino_v1.27.2.ino

amunters committed:
Update README.md

amunters committed:
Add files via upload

Diagnostic program to test some basic hardware functions (tuning pot, switches, touch sensors, etc)

amunters committed:
Update raduino_diag.ino

amunters committed:
Update raduino_diag.ino

amunters committed:
Add files via upload

amunters committed:
Update and rename raduino_v1.27.2.ino to raduino_v1.27.3.ino

amunters committed:
Update README.md

amunters committed:
Update raduino_v1.27.3.ino

amunters committed:
Update raduino_v1.27.3.ino

amunters committed:
Delete raduino_diag.ino

amunters committed:
Add files via upload

diagnostic sketch for raduino

amunters committed:
Update raduino_diag_v3.ino

amunters committed:
Add files via upload

amunters committed:
Update operating-instructions.md

pa28 committed:
The value of c is never changed in printLine, printLine is called with static strings for argument c. Code hygiene suggests this should be declared const char * const c.

pa28 committed:
The change of prototype for printLine removes the need for casting away the constness of static strings.

Replace the empty string with MY_CALLSIGN as a user configurable parameter.

pa28 committed:
Provide for a user parameter callsign, or other text, to be displayed on line 2 when it would otherwise be blank

amunters committed:
Merge pull request #13 from pa28/printLine

Print line

amunters committed:
Update and rename raduino_v1.27.3.ino to raduino_v1.27.4.ino

amunters committed:
Update raduino_v1.27.4.ino

amunters committed:
Update README.md

amunters committed:
Update and rename raduino_v1.27.4.ino to raduino_v1.27.5.ino

amunters committed:
Update README.md

amunters committed:
Add files via upload

amunters committed:
Update operating-instructions.md

amunters committed:
Update and rename raduino_v1.27.5.ino to raduino_v1.27.6.ino

amunters committed:
Update README.md

amunters committed:
Update operating-instructions.md

amunters committed:
Update and rename raduino_v1.27.6.ino to raduino_v1.27.7.ino

amunters committed:
Update README.md

amunters committed:
Add files via upload

amunters committed:
Create library-install.md

amunters committed:
Update README.md

amunters committed:
Update operating-instructions.md

amunters committed:
Create howto.md

amunters committed:
Update and rename howto.md to overview.md

amunters committed:
Create IDE_installation

amunters committed:
Rename IDE_installation to IDE_installation.md

amunters committed:
Update overview.md

amunters committed:
Add files via upload

amunters committed:
Update and rename IDE_installation.md to 1-IDE-installation.md

amunters committed:
Update 1-IDE-installation.md

amunters committed:
Update 1-IDE-installation.md

amunters committed:
Add files via upload

amunters committed:
Add files via upload

amunters committed:
Add files via upload

amunters committed:
Create 2-download-sketch.md

amunters committed:
Update and rename overview.md to 0-overview.md

amunters committed:
Update 0-overview.md

amunters committed:
Create 3-open-sketch.md

amunters committed:
Update 0-overview.md

amunters committed:
Add files via upload

amunters committed:
Create 4-compile-sketch.md

amunters committed:
Update README.md

amunters committed:
Update operating-instructions.md

amunters committed:
Update operating-instructions.md

amunters committed:
Update library-install.md

amunters committed:
Update 0-overview.md

amunters committed:
Update 1-IDE-installation.md

amunters committed:
Update 2-download-sketch.md

amunters committed:
Update 3-open-sketch.md

amunters committed:
Update 4-compile-sketch.md

amunters committed:
Add files via upload

amunters committed:
Create 5-upload-sketch.md

amunters committed:
Update 0-overview.md

amunters committed:
Update 0-overview.md

amunters committed:
Create test.md

amunters committed:
Update 0-overview.md

amunters committed:
Delete 1-IDE-installation.md

amunters committed:
Delete 2-download-sketch.md

amunters committed:
Delete 3-open-sketch.md

amunters committed:
Delete 4-compile-sketch.md

amunters committed:
Delete 5-upload-sketch.md

amunters committed:
Update and rename 0-overview.md to software-installation.md

amunters committed:
Add files via upload

amunters committed:
Rename software-installation.md to  software-installation.md

amunters committed:
Update and rename  software-installation.md to software-installation.md

amunters committed:
Rename software-installation.md to 0-software-installation.md

amunters committed:
Delete test.md

amunters committed:
Update library-install.md

amunters committed:
Update library-install.md

amunters committed:
Delete compile_sketch1.png

amunters committed:
Delete compile_sketch2.png

amunters committed:
Delete compile_sketch3.png

amunters committed:
Delete library-install1.PNG

amunters committed:
Delete library-install2.PNG

amunters committed:
Delete library-install4.PNG

amunters committed:
Delete library-install3.PNG

amunters committed:
Delete library-install5.PNG

amunters committed:
Update README.md

amunters committed:
Update 0-software-installation.md

amunters committed:
Update and rename raduino_v1.27.7.ino to raduino_v1.28.ino

amunters committed:
Update 0-software-installation.md

amunters committed:
Update operating-instructions.md

amunters committed:
Update operating-instructions.md

amunters committed:
Update README.md

amunters committed:
Add files via upload

amunters committed:
Add files via upload

amunters committed:
Update operating-instructions.md

amunters committed:
Update operating-instructions.md

amunters committed:
Update operating-instructions.md

amunters committed:
Add files via upload

amunters committed:
Add files via upload

amunters committed:
Update 0-software-installation (french).md

amunters committed:
Update 0-software-installation (french).md

amunters committed:
Update 0-software-installation (french).md

amunters committed:
Update 0-software-installation (french).md

amunters committed:
Update 0-software-installation (french).md

amunters committed:
Update 0-software-installation (french).md

amunters committed:
Update 0-software-installation (french).md

amunters committed:
Rename 0-software-installation (french).md to 0-instructions d'installation (french)md

amunters committed:
Rename 0-instructions d'installation (french)md to 0-instructions d'installation (french).md

amunters committed:
Rename 0-instructions d'installation (french).md to 0-instructions-d'installation-(french).md

amunters committed:
Rename 0-instructions-d'installation-(french).md to 0-instructions-d'installation-french.md

amunters committed:
Add files via upload

amunters committed:
Update operating-instructions french.md

amunters committed:
Update operating-instructions french.md

amunters committed:
Update operating-instructions french.md

amunters committed:
Update operating-instructions french.md

amunters committed:
Update operating-instructions french.md

amunters committed:
Update operating-instructions french.md

amunters committed:
Create Installer la bibliothèque PinChangeInterrupt

amunters committed:
Rename Installer la bibliothèque PinChangeInterrupt to Installer la bibliothèque PinChangeInterrupt.md

amunters committed:
Update Installer la bibliothèque PinChangeInterrupt.md

amunters committed:
Rename Installer la bibliothèque PinChangeInterrupt.md to Installer la bibliothèque PinChangeInterrupt (french).md

amunters committed:
Update operating-instructions french.md

amunters committed:
Update operating-instructions french.md

amunters committed:
Update operating-instructions french.md

amunters committed:
Rename Installer la bibliothèque PinChangeInterrupt (french).md to installer-la-bibliothèque-PinChangeInterrupt-french.md

amunters committed:
Rename operating-instructions french.md to Instructions utilisateur-french.md

amunters committed:
Update Instructions utilisateur-french.md

amunters committed:
Update Instructions utilisateur-french.md

amunters committed:
Update and rename raduino_v1.28.ino to raduino_v1.28.1.ino

amunters committed:
Update README.md

amunters committed:
Update operating-instructions.md

amunters committed:
Add files via upload

added precompiled binary (hex) file for uploading to Arduino using xLoader

amunters committed:
Update README.md

amunters committed:
Update operating-instructions.md

amunters committed:
Add files via upload

amunters committed:
Delete raduino_v1.28.1.ino

amunters committed:
Delete raduino_v1.28.1.ino.with_bootloader.eightanaloginputs.hex

amunters committed:
Update README.md

amunters committed:
Add files via upload

amunters committed:
Add files via upload

amunters committed:
Delete hardware modification overview.PNG

amunters committed:
Add files via upload

amunters committed:
Update operating-instructions.md

amunters committed:
Add files via upload

amunters committed:
Update operating-instructions.md

corrected pin number for TX-RX wiring mod

amunters committed:
Update Instructions utilisateur-french.md

correction pin number (connecteur P3, broche 16) Cablage TX-RX
